huggingface_datasets_converter/convert.py

```diff
@@ -16,12 +16,17 @@
 TEMPLATE_DATASHEET_PATH = Path(__file__).parent / "datasheet_template.md"
 
 # Mapping from kaggle license identifiers to Hugging Face license identifiers
+# Note: all licenses in datasets inside Kaggle allow re-sharing of datasets; what we are doing here.
+# When license is not specified or is 'other' then re-sharing is not allowed.
 kaggle_license_map = {
     'CC0-1.0': 'cc0-1.0',
     'CC-BY-SA-3.0': 'cc-by-sa-3.0',
     'CC-BY-SA-4.0': 'cc-by-sa-4.0',
     'CC-BY-NC-SA-4.0': 'cc-by-nc-sa-4.0',
     'GPL-2.0': 'gpl-2.0',
+    'GPL-3.0': 'gpl-3.0',
```

Comment:
can you link to where you found these new ones? I don't have access to my actual kaggle acct, only my kaggle.json (because my 2FA is failing because I got a new phone number) so I can't check what shows when I go to "new dataset".

Reply:
Unfortunately, I could not find a webpage where we can find all the licenses available on kaggle. I found that we were missing these from creating a new dataset (screenshot below).

Interestingly, GPL 3.0 no longer appears in the option to choose a license (I might have got it wrong). However, we have two new ones I just added: `lgpl-3.0` and `agpl-3.0`.

![Frame 614](https://user-images.githubusercontent.com/4755430/183659248-336ecd19-007a-4cf3-939a-d2d6b598cb37.png)

I am using [this small notebook](https://gist.github.com/omarespejel/6a466a9cae1254109c3c43e5a6f195a0) to verify the name of the dataset when obtaining the dataset metadata (`data = kaggle.api.metadata_get(user, dataset_name)`).

Reply:
FYI there is also https://www.kaggle.com/general/116302 for common licenses